src/adapters/openai/helpers/completions.ts

```diff
@@ -0,0 +1,79 @@
+                "Your name is : " +
+                botName +
+                "\n" +
+                "Primary Context: " +
+                additionalContext.join("\n") +
+                "\nLocal Context: " +
+                localContext.join("\n"),
+            },
+          ],
+        },
+        {
+          role: "user",
+          content: [
+            {
+              type: "text",
+              text: prompt,
+            },
+          ],
+        },
+      ],
+      temperature: 0.2,
+      max_tokens: MAX_TOKENS,
+      top_p: 0.5,
+      frequency_penalty: 0,
+      presence_penalty: 0,
+      response_format: {
+        type: "text",
+      },
+    });
+    const answer = res.choices[0].message;
+    if (answer && answer.content && res.usage) {
+      return { answer: answer.content, tokenUsage: { input: res.usage.prompt_tokens, output: res.usage.completion_tokens, total: res.usage.total_tokens } };
+    }
+    return { answer: "", tokenUsage: { input: 0, output: 0, total: 0 } };
```

Comment:
Returning an empty string always seems like a bad idea. This seems to make more sense to throw an error